src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -228,6 +230,21 @@ public void handleHelp() {
         }
     }
 
+    /**
+     * Opens the stats window or focuses on it if it's already opened.
+     */
+    @FXML
+    private void handleStats(CommandResult commandResult) {
```

Comment:
name handle -> display stats may be more appropriate
```suggestion
    private void displayStats(CommandResult commandResult) {
```

Reply:
no need to resolve now